scripts/link-validator.conf

```diff
@@ -46,6 +46,14 @@ site-link-validator {
     "https://javadoc.io/static/"
     # GitHub will block with "429 Too Many Requests"
     "https://github.com/"
+    # Github links generated by sbt-license-report
+    "http://github.com/FasterXML/jackson"
```

Comment:
maybe just `http://github.com` to match the `429 Too Many Requests` section above?

Reply:
Fixed and pushed, lets see if this works.